packages/backend/src/models/SavedChartModel.ts

```diff
@@ -359,13 +372,21 @@ export class SavedChartModel {
     async getLatestVersionSummaries(
         chartUuid: string,
     ): Promise<ChartVersionSummary[]> {
+        const getLastVersionUuidSubQuery =
+            this.getLastVersionUuidQuery(chartUuid);
         const chartVersions = await this.getVersionSummaryQuery()
             .where(`${SavedChartsTableName}.saved_query_uuid`, chartUuid)
-            .andWhere(
-                `${SavedChartVersionsTableName}.created_at`,
-                '>=',
-                moment().subtract(60, 'days'),
-            )
+            .andWhere(function () {
+                // get all versions from the last X days + the current version ( in case is older than X days )
+                this.where(
+                    `${SavedChartVersionsTableName}.created_at`,
+                    '>=',
+                    moment().subtract(60, 'days'),
```

Comment:
we can use `current_timestamp - interval '60 days'` here

Reply:
Going to try that in https://github.com/lightdash/lightdash/pull/7450

Reply:
👌